README.md

```diff
@@ -155,3 +158,16 @@ libView.on("searchTextUpdated", function (searchText) {
     return null;
 });
 ```
+## Structure of test 
+- Import libraries required for creating the item tested
+     import { LibraryItem } from '../src/components/LibraryItem';
+- Define a test and verify the results.e.g. for test 
```

Comment:
There are various things that need to be improved from the above:

- The first bullet point is grammatically incorrect
- The `import { }` line shows up incorrect in the resulting page
- The *Define a test...* statement ends unexpectedly with *for test* (also, there needs to be space before `e.g.`)
- The test case body does not relate to `LibraryItem` at all
- Step-by-step illustration (i.e. bullet points) does not lend itself well for this section

I would recommend the section be written like so:

## Structure of a test case
The following code snippet illustrates a general structure of a test case:

```javascript
// Importing the object to be tested
import { LibraryItem } from '../src/components/LibraryItem';

// A test case that tests object properties against their intended value 
describe("LibraryItem tests", function () {
    it("should do bla bla bla", function () {
        // Add some sample LibraryItem tests
    });
});
```

Step by step instructions to author tests can be found on [this wiki page](https://github.com/DynamoDS/librarie.js/wiki/Setting-up-test-frameworks-for-librarie.js).